ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-stable-2.x.yaml

```diff
@@ -56,24 +59,31 @@ tests:
 - as: opendatahub-operator-e2e
   skip_if_only_changed: ^\.github/|^docs/|\.[mM][dD]$|^.gitignore$|^golangci|^crd-ref-docs\.config|^OWNERS$|^PROJECT$|^LICENSE$|^OWNERS_ALIASES$|^Makefile$
   steps:
-    cluster_profile: gcp-opendatahub
+    allow_best_effort_post_steps: true
```

Comment:
do we need to update for "stable-2.x" if the scope in the ticket is only for "rhoai"?

Reply:
i will remove for 2.x